lib/jxl/dec_frame.cc

```diff
@@ -358,6 +358,7 @@ Status FrameDecoder::ProcessDCGlobal(BitReader* br) {
   } else {
     shared.image_features.patches.Clear();
   }
+  shared.image_features.splines = Splines();
```

Comment:
Maybe we want to call a .clear() method that erases all splines but doesn't reallocate the vector? Likely doesn't really matter though

Reply:
Done, thanks.